ldi-orchestrator/ldio-connectors/ldio-http-in-poller/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/LdioHttpInputPoller.java

```diff
@@ -91,6 +91,8 @@ private void executeRequest(Request request) {
 			String content = response.getBodyAsString().orElseThrow();
 			processInput(content, contentType);
 		} else {
+			log.error("Failed to execute request {} {} {}", request.getMethod(), request.getUrl(), response.getHttpStatus());
+			log.atError().log("{}", response.getBodyAsString().orElse(null));
```

Comment:
Why not using `response.getBodyAsString().ifPresent(... actual logging ...)`. I am concerned that you will end up otherwise with an empty logging line like this example below, while you are expecting no logging when the body is `null`:
```
09:07:29.520 [main] INFO LdioHttpInputPoller -- 
```

Reply:
Both log.errors were one line at first, then this made sense but now it doesn't :smile_cat: